airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/common.py

```diff
@@ -18,6 +19,60 @@
 
 logger = logging.getLogger("airbyte")
 
+class SourceFacebookMarketingConfig:
```

Comment:
this feels like re-inventing the wheel, is there any particular reason to replace pydantic with this?

Reply:
See thread in slack: https://airbytehq.slack.com/archives/C019WEENQRM/p1636488040184300

Summary: I am unable to create a "oneOf" type structure using pydantic. Any help welcome.

Reply:
I see, this PR should definitely help https://github.com/airbytehq/airbyte/pull/8485